pkg/utils/config/address_setting_compare.go

```diff
@@ -0,0 +1,2164 @@
+			if *curSetting.ConfigDeleteAddresses != *newSetting.ConfigDeleteAddresses {
+				return false
+			}
+		} else {
+			return false
+		}
+		if newSetting.ManagementBrowsePageSize == nil {
+			if curSetting.ManagementBrowsePageSize != nil {
+				return false
+			}
+		} else if curSetting.ManagementBrowsePageSize != nil {
+			if *curSetting.ManagementBrowsePageSize != *newSetting.ManagementBrowsePageSize {
+				return false
+			}
+		} else {
+			return false
+		}
+		if newSetting.DefaultPurgeOnNoConsumers == nil {
+			if curSetting.DefaultPurgeOnNoConsumers != nil {
+				return false
+			}
+		} else if curSetting.DefaultPurgeOnNoConsumers != nil {
+			if *curSetting.DefaultPurgeOnNoConsumers != *newSetting.DefaultPurgeOnNoConsumers {
+				return false
+			}
+		} else {
+			return false
+		}
+		if newSetting.DefaultMaxConsumers == nil {
+			if curSetting.DefaultMaxConsumers != nil {
+				return false
+			}
+		} else if curSetting.DefaultMaxConsumers != nil {
+			if *curSetting.DefaultMaxConsumers != *newSetting.DefaultMaxConsumers {
+				return false
+			}
+		} else {
+			return false
+		}
+		if newSetting.DefaultQueueRoutingType == nil {
+			if curSetting.DefaultQueueRoutingType != nil {
+				return false
+			}
+		} else if curSetting.DefaultQueueRoutingType != nil {
+			if *curSetting.DefaultQueueRoutingType != *newSetting.DefaultQueueRoutingType {
+				return false
+			}
+		} else {
+			return false
+		}
+		if newSetting.DefaultAddressRoutingType == nil {
+			if curSetting.DefaultAddressRoutingType != nil {
+				return false
+			}
+		} else if curSetting.DefaultAddressRoutingType != nil {
+			if *curSetting.DefaultAddressRoutingType != *newSetting.DefaultAddressRoutingType {
+				return false
+			}
+		} else {
+			return false
+		}
+		if newSetting.DefaultConsumerWindowSize == nil {
+			if curSetting.DefaultConsumerWindowSize != nil {
+				return false
+			}
+		} else if curSetting.DefaultConsumerWindowSize != nil {
+			if *curSetting.DefaultConsumerWindowSize != *newSetting.DefaultConsumerWindowSize {
+				return false
+			}
+		} else {
+			return false
+		}
+		if newSetting.DefaultRingSize == nil {
+			if curSetting.DefaultRingSize != nil {
+				return false
+			}
+		} else if curSetting.DefaultRingSize != nil {
+			if *curSetting.DefaultRingSize != *newSetting.DefaultRingSize {
+				return false
+			}
+		} else {
+			return false
+		}
+		if newSetting.RetroactiveMessageCount == nil {
+			if curSetting.RetroactiveMessageCount != nil {
+				return false
+			}
+		} else if curSetting.RetroactiveMessageCount != nil {
+			if *curSetting.RetroactiveMessageCount != *newSetting.RetroactiveMessageCount {
+				return false
+			}
+		} else {
+			return false
+		}
+		if newSetting.EnableMetrics == nil {
+			if curSetting.EnableMetrics != nil {
+				return false
+			}
+		} else if curSetting.EnableMetrics != nil {
+			if *curSetting.EnableMetrics != *newSetting.EnableMetrics {
+				return false
+			}
+		} else {
+			return false
+		}
+	}
+	return true
+}
```

Comment:
Right, I noticed that the only func actually used in the v1beta1 compare for address settings. I didn't want to rock the boat with the address settings, just rename the file since config.go would have grown too large with acceptor & connector comparison functions.